lib/galaxy/webapps/base/webapp.py

```diff
@@ -195,6 +195,7 @@ def create_mako_template_lookup(self, galaxy_app, name):
             )
 
     def handle_controller_exception(self, e, trans, method, **kwargs):
+        log.exception(f"Encountered exception in controller method: {method}")
```

Comment:
```suggestion
        log.debug(f"Encountered exception in controller method: {method}", exc_info=True)
```
and that's a concession here, it should really not be logged IMO.

Reply:
As a dev I would love also - I am always adding an exception handler to each API endpoint I'm working. I lack your vscode skills. What if we didn't log if ``e`` is a MessageException with ``e.status_code < 500`` - and logged all the other exceptions at a high level than debug? Shouldn't all 500 and unknown exceptions be logged?

Reply:
but your client surely blows up with the exception message? I rarely use the debugger for anything other than debugging ?

Reply:
(and they do for >=500, @dannon added that recently)

Reply:
I trust you and I'm glad we do it then. In my defense - I literally just decorated an API endpoint this week to test. Maybe it was for pydantic modeling stuff that came through as a sort unexpected-to-me 4XX though. Sorry for the noise.

Reply:
I think that's a slightly different issue (and less likely to happen in the WSGI app here). I agree these are harder to debug, I think there are various ways in which we could improve validation errors in `validation_error_to_message_exception`. Next time there's an unparsable validation error we should take a closer look.